scripts/check-install-scripts.js

```
@@ -0,0 +1,77 @@
+      try {
+        const pkg = JSON.parse(fs.readFileSync(pkgPath, 'utf8'));
+        const scripts = [];
+        if (pkg.scripts) {
+          if (pkg.scripts.preinstall) scripts.push('preinstall');
+          if (pkg.scripts.install) scripts.push('install');
+          if (pkg.scripts.postinstall) scripts.push('postinstall');
+        }
+        if (scripts.length > 0 && !allowed.has(pkg.name)) {
+          foundScripts.push({ name: pkg.name, scripts });
+        }
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)